.docker/scripts/init-and-run.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash -x
+
+# Turn on monitor mode, required by job control
+set -m
+
+if [ ! -f /var/www/html/magento2/.installed ]; then
+
+  until mysql -u${MYSQL_USER} -p${MYSQL_PASSWORD} -h${MYSQL_HOST}; do
+    >&2 echo "MySQL is unavailable - sleeping"
+    sleep 5
+  done
+  cd /var/www/html/magento2/
+  composer require nosto/php-sdk:@stable
```

Comment:
Why is this needed? Where is the actual plugin mounted?

Reply:
app/code/Nosto/Tagging.
If we don't have the SDK on Magento, it won't find the references and it won't work